brave/brave-core
# Harden the import code for Brave Payments data

Fixes https://github.com/brave/brave-browser/issues/2312

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:
- Run steps to reproduce in https://github.com/brave/brave-browser/issues/2312
- Import should work

## Reviewer Checklist:
